### PR TITLE
SEN-1453 -- create repo and first commit for @cascadeenergy/config-client

### PR DESCRIPTION
**Pull request by notxt**
**Testing:**
- add a key/value to your consul key store
- `node examples/test.js [internal consul ip] [path to test key]`
- your key should print out

**Ticket:**
https://cascadeenergy.atlassian.net/browse/SEN-1453

### CODE REVIEW

**Comment by notxt** [comment]
@nackjicholson https://github.com/CascadeEnergy/consul-kv/pull/2 got messed up during rename so I closed and reopened a new PR

**Comment by nackjicholson** [comment]
If you want to put your testing script in an `examples` directory go for it, you'll have to change the particulars...but yeah then it's there for people.

Also you could put it in the README to help people understand what this does.

File: es6/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import got from 'got-promise';
+
+function pluckValue(response) {
+  return response.body[0].Value;
```
Comment (nackjicholson) [inline es6/index.js]: Any reason we wouldn't want the whole object back? `response.body[0]`. Not sure really what the rest of that response looks like, but I assume Consul has a reason for exposing it to you when you get a key.

Comment (notxt) [inline es6/index.js]: just that I wanted to treat it like a simple key/value store and ignore everything else

- [ ] return entire consul response

**Comment by notxt** [comment]
- [ ] add example code
- [ ] add readme

**Comment by notxt** [comment]
@nackjicholson good to go

**Comment by nackjicholson** [comment]
:ok: :shipit:

**Pull request merged**